de.metas.shipper.gateway.dhl/src/main/java/de/metas/shipper/gateway/dhl/model/DhlCustomDeliveryData.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * #%L
+ * de.metas.shipper.gateway.dhl
+ * %%
+ * Copyright (C) 2019 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.shipper.gateway.dhl.model;
+
+import com.google.common.collect.ImmutableMap;
+import de.metas.shipper.gateway.spi.model.CustomDeliveryData;
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder(toBuilder = true)
+public class DhlCustomDeliveryData implements CustomDeliveryData
+{
+	ImmutableMap<Integer, String> packageIdsToSequenceNumber;
+	ImmutableMap<String, byte[]> sequenceNumberToPdfLabel;
+	ImmutableMap<String, String> sequenceNumberToAWB;
```

Comment:
1. Avoid having such maps in getters. It would be better to have some methods like getSequenceNumberByPackageId(..)
2. Avoid using primitive-ish types as keys (e.g. Integer, String etc) in Maps.